### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by JanZachmann**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,19 +8,20 @@ license = "MIT OR Apache-2.0"
...
-version = "0.20.1"
+version = "0.20.2"
 
 [dependencies]
 actix-server = "2.3"
 actix-web = "4.5"
 anyhow = "1.0"
 async-trait = "0.1"
-azure-iot-sdk = { git = "https://github.com/omnect/azure-iot-sdk.git", tag = "0.13.0", features = [
+azure-iot-sdk = { git = "https://github.com/omnect/azure-iot-sdk.git", tag = "0.13.1", features = [
```
Comment (JanZachmann) [inline Cargo.toml:18]: @JanZachmann  re-build when available

File: src/twin/network_status.rs [hunk 2]
```diff
@@ -188,17 +188,18 @@ impl NetworkStatus {
...
-                        .addr_v4
-                        .get_or_insert(vec![])
-                        .push(addr.ip.to_string()),
-                    Some(Addr::V6(addr)) => entry
-                        .addr_v6
-                        .get_or_insert(vec![])
-                        .push(addr.ip.to_string()),
-                    None => error!("report_network_status: ip address is missing"),
-                };
+                for a in &i.addr {
```
Comment (mlilien) [inline src/twin/network_status.rs:191]: i guess that's a change which has nothing to do with the dependency updates and is a fix for publishing all ip's. imho should be reflected in PR description.

Comment (JanZachmann) [reply src/twin/network_status.rs:191]: no, it is a result of the update

File: .cargo/config.toml [hunk 3]
```diff
@@ -1,2 +1,3 @@
 [env]
-RUST_TEST_THREADS = "1"
\ No newline at end of file
+RUST_TEST_THREADS = "1"
+SOCKET_DIR = { value = "/run/aziot", force = false }
```
Comment (empwilli) [inline .cargo/config.toml:3]: Where does the SOCKET_DIR variable come from and why do we need it at build time?

**Review by empwilli — CHANGES_REQUESTED** [review]

**Review by mlilien — APPROVED** [review]

File: .cargo/config.toml (hunk 3 above)
Comment (JanZachmann) [reply .cargo/config.toml:3]: it come from eis-utils dependency which now makes use of azure iot-identity-service 1.5.0 which requires that compile time envvar

Comment (empwilli) [reply .cargo/config.toml:3]: I see, can we maybe add a comment so that we don't forget about that?

Comment (JanZachmann) [reply .cargo/config.toml:4]: done

**Pull request merged**